Yoast/wordpress-seo
# Changed .nomin.js to js and .js to .min.js

## Summary

This PR can be summarized in the following changelog entry:

* Fixes an error where the chunk files where not added to the `.zip` file on release.

## Relevant technical choices:

* Added all files in `/js/dist/chunks/` to commit.

## Test instructions

This PR can be tested by following these steps:

* Build the project
* Run `grunt artifact`
* Make sure the chunks folder is added to the `artifacts/js/dist` folder.

## Quality assurance


Fixes #8822
